populous/generators/date.py

```diff
@@ -43,7 +45,7 @@ def generate(self):
             start = to_timestamp(datetime.now())
         else:
             # try not to go too far in the past
-            start = to_timestamp(datetime(1900, 1, 1))
+            start = to_timestamp(datetime(self.epoch_year, 1, 1))
```

Comment:
So if I understand correctly, on Mac OS `mktime` with a negative value raises an error.

The docs do say:
> If the input value cannot be represented as a valid time, either OverflowError or ValueError will be raised (which depends on whether the invalid value is caught by Python or the underlying C libraries). The earliest date for which it can generate a time is platform-dependent.

So ok ¯\_(ツ)_/¯